packages/integrations/vercel/README.md

```diff
@@ -238,9 +238,11 @@ export default defineConfig({
 
 ### Function bundling configuration
 
-The Vercel adapter splits builds into a separate function per route by default. This helps reduce the size of each function, as it only bundles code used on that page.
+The Vercel adapter combines all of your routes into a single function by default.
 
-You can disable this and build to a single function by setting the `functionPerRoute` configuration option to `false`:
+You also have the option to split into a seperate function for each route using the `functionPerRoute` option. Doing so means your functions are smaller, you're less likely to exceed the size limit, and code starts are faster. 
```

Comment:
```suggestion
You also have the option to split builds into a separate function for each route using the `functionPerRoute` option. This reduces the size of each function, meaning you are less likely to exceed the size limit for an individual function. Also, code starts are faster. 
```
I added a bit because people will skim and confuse **size of a function** with "size = NUMBER of functions"... and they are totally at risk of exceeding THAT limit.

Reply:
ah yeah good point.